docker-compose.yml

```
@@ -0,0 +1,88 @@
+version: '2.3'
+
+x-volumes:
+  &default-volumes
+    volumes:
+      - files:/app/web/sites/default/files
+
+x-environment:
+  &default-environment
+    LAGOON_ROUTE: ${LOCALDEV_URL:-http://govcms8.docker.amazee.io}
+
+volumes:
+  files: {}
+
+services:
+
+  cli: # cli container, will be used for executing composer and any local commands (drush, drupal, etc.)
+    build:
+      context: .
+      dockerfile: $PWD/.docker/Dockerfile.govcms8
+    image: ${DOCKERHUB_NAMESPACE:-govcms}/govcms8
+    << : *default-volumes # loads the defined volumes from the top
+    environment:
+      << : *default-environment
+
+  test:
+    build:
+      context: .
+      dockerfile: $PWD/.docker/Dockerfile.test
+      args:
+        CLI_IMAGE: ${DOCKERHUB_NAMESPACE:-govcms}/govcms8
+    image: ${DOCKERHUB_NAMESPACE:-govcms}/test
+    << : *default-volumes
+    depends_on:
+      - cli
+    environment:
+      << : *default-environment
+
+  nginx:
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile.nginx-drupal
+      args:
+        CLI_IMAGE: ${DOCKERHUB_NAMESPACE:-govcms}/govcms8
```

```suggestion
        CLI_IMAGE: ${DOCKERHUB_NAMESPACE:-govcms8}/nginx-drupal
```
places in correct default docker hub namespace and names container correctly